clientupdate/clientupdate.go

```diff
@@ -831,6 +833,19 @@ func (up *Updater) installMSI(msi string) error {
 	return err
 }
 
+func (up *Updater) cleanupOldDownloads(glob string) {
+	matches, err := filepath.Glob(glob)
+	if err != nil {
+		up.Logf("cleaning up old downloads: %v", err)
+		return
+	}
+	for _, m := range matches {
+		if err := os.RemoveAll(m); err != nil {
```

Comment:
what if I make a symlink in /tmp/ named /tmp/tailscale-update-foo pointing to `/`? will this delete the whole world recursively as root?